R/zzz.R

```diff
@@ -4,10 +4,6 @@
   if (getRversion() >= "2.15.1") {
```

Comment:
Do we need this if test when we anyway require a newer R version?

Reply:
Fixed in #127.